reconcile/skupper_network/models.py

```diff
@@ -0,0 +1,291 @@
+from __future__ import annotations
+
+import hashlib
+import logging
+from collections.abc import Iterable
+from typing import (
+    Any,
+    Optional,
+)
+
+from pydantic import (
+    BaseModel,
+    Field,
+)
+
+from reconcile.gql_definitions.skupper_network.skupper_networks import (
+    ClusterPeeringConnectionClusterAccepterV1,
+    ClusterPeeringConnectionClusterRequesterV1,
+    ClusterV1,
+    NamespaceV1,
+    SkupperSiteConfigDefaultsV1,
+    SkupperSiteConfigV1,
+)
+
+
+class Defaults:
+    """Define default values for a SkupperSite model."""
```

Comment:
Why do we query `siteConfigDefaults` from gql server if they are defined here? I see they are all merged together in `init()`, but I'm not sure I understand why we need to define defaults in 2 places?

Reply:
Via `siteConfigDefaults`, you can define defaults used by all skupper sites, e.g., `routers: 1`, if not set via `siteConfigDefaults` or via skupper config in the namespace the `models.py Default` is being used.